libfreerdp/core/fastpath.c

```diff
@@ -466,7 +483,7 @@ static int fastpath_recv_update_data(rdpFastPath* fastpath, wStream* s)
 			if ((fastpath->fragmentation != FASTPATH_FRAGMENT_FIRST) &&
 					(fastpath->fragmentation != FASTPATH_FRAGMENT_NEXT))
 			{
-				WLog_ERR(TAG,  "Unexpected FASTPATH_FRAGMENT_LAST");
+				WLog_ERR(TAG, "fastpath_recv_update_data: Unexpected FASTPATH_FRAGMENT_LAST");
```

Comment:
To many work by hands, very long lines with code:
WLOG_ERR(TAG, "Unexpected FASTPATH_FRAGMENT_LAST");

Also: __FUNCTION": "_fmt - calculated at build.
Also WLOG_ERR in any time can be modified to add line num or other info, or format may be changed.